src/azul/indexer/document.py

```diff
@@ -1091,6 +1091,13 @@ class VersionType(Enum):
 
 InternalVersion = tuple[int, int]
 
+
+class OpType(Enum):
+    index = auto()
+    create = auto()
+    delete = auto()
```

Comment:
```suggestion
class OpType(Enum):
    #: Write the document to the index, overwriting it if it already exists
    index = auto()
    
    #: Write the document to the index or fail if it already exists
    create = auto()
    
    # Remove the document from the index or fail if it does not exist
    delete = auto()
```

But make sure these descriptions are accurate.

Reply:
They are accurate.